pkg/export/setup/setup.go

```diff
@@ -50,6 +50,9 @@ const (
 	UAModeUnspecified = "unspecified"
 	UAModeAVMW        = "on-prem"
 	UAModeABM         = "baremetal"
+	UAModeAWS         = "aws"
+	UAModeAzure       = "azure"
+	UAModeAttached    = "attached"
```

Comment:
What is "attached"?

Reply:
[Attached clusters](https://cloud.google.com/anthos/clusters/docs/multi-cloud/attached) could be any k8s cluster that customers want to connect to Google.

Reply:
Gotcha - could you add a comment?